tests/vdo_test.py

```diff
@@ -277,6 +278,20 @@ def test_grow_physical(self):
         self.assertEqual(info_before.logical_size, info_after.logical_size)
         self.assertGreater(info_after.physical_size, info_before.physical_size)
 
+    def test_statistics(self):
+        """Verify that it is possible to retrieve statistics of an existing VDO volume"""
+
+        ret = BlockDev.vdo_create(self.vdo_name, self.loop_dev)
+        self.addCleanup(self._remove_vdo, self.vdo_name)
+        self.assertTrue(ret)
+
+        with six.assertRaisesRegex(self, GLib.GError, "No such file or directory"):
+            stats = BlockDev.vdo_get_statistics("nonexistingxxx")
+
+        stats = BlockDev.vdo_get_statistics(self.vdo_name)
+        self.assertIsNotNone(stats)
+        self.assertGreater(len(stats), 0)
+
```

Comment:
We should check at least some of the returned data to be sure it is the same what vdo tools report.